lib/index.ts

```diff
@@ -24,9 +24,17 @@ export async function percySnapshot(page: Page, name: string, options: any = {})
   if (!name) {
     throw new Error("'name' must be provided. In Mocha, this.test.fullTitle() is a good default.")
   }
-  await page.addScriptTag({
-    path: agentJsFilename()
-  })
+
+  try {
+    await page.addScriptTag({
+      path: agentJsFilename()
+    })
+  } catch (err) {
+    // Certain CSP settings prevent Puppeteer from injecting scripts. See:
+    // https://github.com/GoogleChrome/puppeteer/issues/2644
+    console.log(`[percy] Could not snapshot, maybe due to stringent CSPs. Try page.setBypassCSP(true).`)
```

Comment:
This prints to the customer's console every time this happens. I think this should be rare enough that it is nice to show a message explaining what's going on, but let me know if you think this seems too noisy.